manifest.toml

```diff
@@ -0,0 +1,54 @@
+packaging_format = 2
+
+id = "phpboost"
+name = "PHPBoost"
+description.en = "French and free content management system (CMS)"
+description.fr = "système de gestion de contenu (CMS) français et libre"
+
+version = "5.2.11~ynh1"
+
+maintainers = ["liberodark"]
+
+[upstream]
+license = "GPL-3.0-only"
+website = "https://www.phpboost.com/"
+demo = "https://demo.phpboost.com/"
+admindoc = "https://www.phpboost.com/wiki/wiki.php"
+code = "https://github.com/PHPBoost/PHPBoost"
+
+[integration]
+yunohost = ">= 11.1.11"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+[install]
+    [install.ports]
```

Comment:
Eeeeh I guess you meant it as a resource ?

Reply:
ah yes...